samples/cancel-training-pipeline.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+async function main(
+    trainingPipelineId,
+    project,
+    location = 'us-central1',
+) {
+  // [START aiplatform_cancel_training_pipeline]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.\
+   * (Not necessary if passing values as arguments)
+   */
+
+  // const trainingPipelineId = 'YOUR_PIPELINE_ID';
+  // const project = 'YOUR_PROJECT_ID';
+  // const location = 'YOUR_PROJECT_LOCATION';
+
+  // Imports the Google Cloud Pipeline Service Client library
+  const {PipelineServiceClient} = require('@google-cloud/aiplatform');
+
+  // Specifies the location of the api endpoint
+  const clientOptions = {
+    apiEndpoint: 'us-central1-aiplatform.googleapis.com',
+  };
+
+  // Instantiates a client
+  const pipelineServiceClient = new PipelineServiceClient(clientOptions);
+
+  async function cancelTrainingPipeline() {
+    // Configure the resource
+    const name = pipelineServiceClient.trainingPipelinePath(
+        project,
+        location,
+        trainingPipelineId,
+    );
+    const request = {
+      'name': name,
+    };
+
+    // Cancel training pipeline request
+    const [response] = await pipelineServiceClient.cancelTrainingPipeline(request);
+
+    console.log(`Cancel training pipeline response :`);
+  }
+  // [END aiplatform_cancel_training_pipeline]
+  await cancelTrainingPipeline();
+}
+
+main(...process.argv.slice(2)).catch((err) => {
+  console.error(err);
+  process.exitCode = 1;
+});
```

Comment:
Please do a run through and add newlines to end of all these files so we're not fighting the great (-)

Reply:
yeah, I just ran prettier against the whole thing, and it got all these.